mm2src/coins/lp_coins.rs

```diff
@@ -1410,25 +1486,178 @@ pub trait CoinAssocTypes {
     fn parse_signature(&self, sig: &[u8]) -> Result<Self::Sig, Self::SigParseError>;
 }
 
-/// Operations specific to the [Trading Protocol Upgrade implementation](https://github.com/KomodoPlatform/komodo-defi-framework/issues/1895)
+pub struct SendMakerPaymentArgs<'a, Coin: CoinAssocTypes + ?Sized> {
+    /// Maker will be able to refund the payment after this timestamp
+    pub time_lock: u64,
```

Comment:
`time_lock` is a common name over the entire codebase, it's used in many wrapper structures and fn arguments. So, if we change it here, it might be preferred to change it everywhere else 🙂 Also, I will likely add `time_lock_duration` field when adding other coin protocols to new swaps. Will it help to avoid confusion?

Reply:
time_lock_duration helps little bit more, but still, to understand the format of the duration we need to check where these fields are used. But you don't need to worry about that in this PR I think.

> So, if we change it here, it might be preferred to change it everywhere else

What do you think about creating an issue for community with "help-needed" label? This kind of easy-to-fix (at the end this is just updating bunch of field names in the project) issues can be good for getting contributions from the community.

Reply:
Done https://github.com/KomodoPlatform/komodo-defi-framework/issues/2054